aws-android-sdk-cognitoauth/src/main/java/com/amazonaws/mobileconnectors/cognitoauth/util/Pkce.java

```diff
@@ -48,7 +49,7 @@ public final static String generateRandom() {
     public final static String generateHash(String data) throws Exception {
         String hashedData = data;
         try {
-            byte[] bytes = data.getBytes("US-ASCII");
+            byte[] bytes = data.getBytes(StandardCharsets.US_ASCII);
```

Comment:
Unfortunately, we cannot use this API as it was introduced in API level 19. Our SDK supports platforms that are lower than 19. Can you revert this back to "US-ASCII" string? We are reviewing the Base64 encoding change,